applications/app/controllers/AtomPageController.scala

```diff
@@ -0,0 +1,64 @@
+package controllers
+
+import com.gu.contentapi.client.model.v1.ItemResponse
+import common._
+import model.Cached.{RevalidatableResult, WithoutRevalidationResult}
+import model._
+import play.api.mvc._
+import play.twirl.api.Html
+import scala.concurrent.Future
+import contentapi.ContentApiClient
+import model.content._
+import views.html.fragments.atoms.{ media => MediaAtomBody, storyquestions => StoryQuestionsAtomBody }
+
+class AtomPageController(contentApiClient: ContentApiClient)(implicit context: ApplicationContext) extends Controller with Logging with ExecutionContexts {
+
+  val notYetImplemented = (NotFound, Nil)
+
+  def render(atomType: String, id: String, js: Boolean) = Action.async { implicit request =>
```

Comment:
`js` is quite a generic name. Would something like `isJavascriptEnabled` or similar be better?